src/code/z_common_data.c

```diff
@@ -4,7 +4,7 @@ SaveContext gSaveContext;
 
 void SaveContext_Init(void) {
     bzero(&gSaveContext, sizeof(gSaveContext));
-    gSaveContext.playerForm = 0;
+    gSaveContext.playerForm = PLAYER_FORM_FIERCE_DEITY;
```

Comment:
How playerForm treats a value of 0 is a special case, and will need more discussion on how to handle it. It doens't make sense for the player to be initialized as fierce deity